### ISSUE TITLE
CNNIQA output range

### ISSUE THREAD

**Issue by hazelAutumn** [chaofengc/IQA-PyTorch#70]
Hi,
Thank you so much for your awesome work.
I have a question about the pretrained CNNIQA in this repo. As I understood, this model was trained on KonIQ in this repo, and the output range should be in [0, 100] as the zMOS provided in the dataset. However, when I used the pretrained metric in this repo, I see that some score outputs are negative, for examples: -4.12, or -3.33.
Could you please give the specific output range for this pretrained model?
Thank you so much!

**Comment by chaofengc** [comment]
Thanks for interest. Please note that there are no functions to constrain the output of CNNIQA model. Therefore, it is expected that there are some values out of [0, 100], although the model is trained on KonIQ.

The model outputs can only indicate higher quality with higher scores.
